libmscore/style.cpp

```diff
@@ -543,6 +545,7 @@ static const StyleType styleTypes[] {
       { StyleIdx::tupletFontBold,                "tupletFontBold",               false },
       { StyleIdx::tupletFontItalic,              "tupletFontItalic",             true },
       { StyleIdx::tupletFontUnderline,           "tupletFontUnderline",          false },
+      { StyleIdx::tupletAlign,                   "tupletAlign",                  QVariant::fromValue(Align::CENTER) },
```

Comment:
Don't have to do anything with this PR indeed - don't know, what this means; at least I didn't include it deliberately ...